packages/core/src/charts/meter.ts

```diff
@@ -28,13 +28,20 @@ export class MeterChart extends Chart {
 	constructor(holder: Element, chartConfigs: ChartConfig<MeterChartOptions>) {
 		super(holder, chartConfigs);
 
+		let options = chartConfigs.options;
+		// check if user has added proportional options
+		if (chartConfigs.options.meter.proportional) {
+			options = Tools.merge(
+				Tools.clone(Configuration.options.meterChart),
+				{ legend: { enabled: true } },
```

Comment:
Why not have a `Configuration.options.proportionalMeterChart` that extends `Configuration.options.meterChart` in `Configuration.ts` & adds the legend config?